packages/rancher-backup-crd/package.yaml

```diff
@@ -0,0 +1,3 @@
+url: https://github.com/rancher/backup-restore-operator/releases/download/v1.0.3/rancher-backup-crd-1.0.3.tgz
+packageVersion: 01
+releaseCandidateVersion: 02
```

Comment:
We should bump the rancher-backup version to 1.0.4 and switch the packageVersion and releaseCandidateVersion to 00 and 00 to be in sync with other charts of a similar nature (e.g. fleet stuff today).

https://github.com/rancher/charts/blob/ead512d870063ab3164c7484795f8c9dd0f5b599/packages/fleet-agent/package.yaml#L2-L3

Reply:
Agreed.